src/main/java/tests/uiTests/ShouldDisplayDirectoryOfRepresentativesTextTest.java

```diff
@@ -0,0 +1,22 @@
+package tests.uiTests;
+
+import domain.pages.DirectoryPage;
+import domain.pages.HomePage;
+import org.junit.Test;
+import org.openqa.selenium.support.PageFactory;
+import tests.uiTests.UITestBaseClass;
+
+public class ShouldDisplayDirectoryOfRepresentativesTextTest extends UITestBaseClass {
+    /**
+     * Go to representatives
+     * Validate if directory of representatives text displayed.
+     */
+    @Test
+    public void isDirectoryOfRepresentativesDisplayedTest() {
+        HomePage homePage = new PageFactory().initElements(driver, HomePage.class);
+        homePage.clickRepresentativesLink();
+        DirectoryPage directoryPage = new PageFactory().initElements(driver, DirectoryPage.class);
+        assert directoryPage.isDisplayedDirectoryOfRepresentativesText();
```

Comment:
use static methods from Assert class of JUINT

Reply:
done